pkg/server/http/http.go

```diff
@@ -30,16 +30,20 @@ type Server struct {
 }
 
 // New creates a new Server.
-func New(logger log.Logger, reg *prometheus.Registry, comp component.Component, prober *prober.HTTPProbe, opts ...Option) *Server {
+func New(logger log.Logger, reg *prometheus.Registry, comp component.Component, webRoutePrefix string, prober *prober.HTTPProbe, opts ...Option) *Server {
 	options := options{}
 	for _, o := range opts {
 		o.apply(&options)
 	}
 
+	if webRoutePrefix == "/" {
```

Comment:
We can skip that if we would just use `path.Join` instead of `+` in `registerProfiler` (:

Reply:
Thanks! will update.